packages/storybook/src/nimble/chip/chip.mdx

```diff
@@ -36,6 +36,19 @@ is:
 
```

Comment:
It'd be good to think through the feedback from the demo meeting about the difference between a selectable (non-removable) chip and a toggle button. What use cases is each one best suited for? Then we could add those docs to each component and consider cross linking them.